src/api/handlers/user.ts

```diff
@@ -0,0 +1,40 @@
+import { RequestManager } from "../../rest/request_manager.ts";
+import { CreateGuildPayload, UserPayload } from "../../types/mod.ts";
+import { endpoints } from "../../util/constants.ts";
+
+/**
+ * Returns the user object of the requester's account.
+ * For OAuth2, this requires the `identify` scope, which will return the object without an email,
+ * and optionally the email scope, which returns the object with an email.
+ */
+export function getCurrentUser() {
+  return RequestManager.get(endpoints.USER_BOT) as Promise<
+    UserPayload
+  >;
+}
+
+/**
+ * Returns a list of partial guild objects the current user is a member of. 
+ * Requires the `guilds` OAuth2 scope.
+ * 
+ * ⚠️ **If you need this, you are probably doing something wrong. Always use cache.guilds
+ *
+ * Advanced Devs:
+ * This function gets you a list of all guilds in those your bot is in.
+ */
+export function getCurrentUserGuilds() {
+  return RequestManager.get(endpoints.USER_GUILDS) as Promise<
+    Partial<CreateGuildPayload>
+  >;
+}
+
+/** Returns a list of connection objects. Requires the `connections` OAuth2 scope. */
+export function getCurrentUserConnections() {
+  // TODO(itohatweb): types: better return type
+  return RequestManager.get(endpoints.USER_CONNECTIONS);
+}
+
+/** This function will return the raw user payload in the rare cases you need to fetch a user directly from the API. */
+export function getUser(userID: string) {
```

Comment:
Breaking change, why? We are just moving this function from `guild` handler to a new handler `user`.

Reply:
Not everyone will import from `mod.ts` moving this could be considerd a breaking change if someone is importing from say https://deno.land/x/discordeno@10.1.0/src/api/handlers/guild.ts

Reply:
@Skillz4Killz, it is NOT RECOMMENDED TO or you SHOULD NOT export functions directly from a specific file, but always export an entity from the main file, that is, `mod.ts` conventionally.